.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,50 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check-pr-title:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title and Manage Review
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const title = context.payload.pull_request.title;
+            const pattern = /^(feat|fix|build|chore|ci|docs|perf|refactor|revert|style|test):\s\w+:\s[a-z].+/;
```

Comment:
(just cluing into this now while thinking about it fresh) I'm curious why we deviate from conventional commit style: https://www.conventionalcommits.org/en/v1.0.0/ ?

We do:

``feat: api: description``

Conventional commit has the "scope" with "()":

``feat(api): description``

---
Also, nit, but maybe alphabetize the "types"?

``build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test``